tests/Unit/Anonymization/Configuration/AttributeLoaderTest.php

```diff
@@ -68,6 +67,43 @@ public function testLoadOk(): void
         self::assertSame('country', $testTableConfig['address_0']->options->get('country'));
     }
 
+    public function testLoadWithJoinedInheritance(): void
+    {
+        $classMetadataFactory = $this->getClassMetadataFactory();
+        $classMetadataFactory->getMetadataFor(TestEntityWithEmbedded::class);
```

Comment:
Is it normal to have `TestEntityWithEmbedded` here?

Reply:
I guess it's not, will fix it.